tools/python_api/src_cpp/include/cached_import/py_cached_modules.h

```diff
@@ -82,6 +82,13 @@ class PandasCachedItem : public PythonCachedItem {
     PythonCachedItem NaT;
 };
 
+class PolarsCachedItem : public PythonCachedItem {
+public:
+    PolarsCachedItem() : PythonCachedItem("polars"), DataFrame("DataFrame", this) {}
+
+    PythonCachedItem DataFrame;
```

Comment:
What's our naming convention? Shouldn't we keep it consistent with cpp code under `src/`? I saw `DataFrame`, `series`, and `from_dict` in this file, which is quite confusing. Let's clean it up and make it consistent with our `src` code.

Reply:
The naming is made to be identical to how you would access the module/attribute from python. For example, `pandas.DataFrame` would be accessed via `importCache->pandas.DataFrame()`.